homeworkreal4/LogicPractice.java

```diff
@@ -0,0 +1,14 @@
+package homeworkreal4;
+
+public class LogicPractice {
+
+    public static void main(String[] args) {
+
+        Boolean p = true;
+        Boolean q = false;
+        Boolean r = true;
+
+        System.out.println(p || r && !q);
```

Comment:
Good effort, but your translation is incorrect.  The exercise 1 group 1 question: `p ⋀ ¬r ⋁ ¬q`

Math | Java
-------|-------
⋀|&&
⋁ | \|\|

When you are learning and trying to replicate it, always go back to the source and step through your work.  Compare what you have done with the information you are learning from so you can verify your understanding and catch and correct your own mistakes, if any.  For instance, taking a  moment to compare the symbols you have chosen for ⋀ (AND) and ⋁ (OR) with slides from https://docs.google.com/presentation/d/1y3Q6OB4u4u02d7x6ZFqZ67L_y-bVz4KTcUi2VpQP8Yc/edit?usp=sharing (also linked int the homework) would have led you to realize that the wrong Java logic operators were used.

Always take it step-by-step, character by character.